pkg/cmd/util/helpers.go

```diff
@@ -149,6 +151,34 @@ func RetrieveApplicationsByName(ctx context.Context, c client.Client, applicatio
 	return applicationMetaList, nil
 }
 
+func RetrieveApplicationStatusByName(ctx context.Context, c client.Client, applicationName string, namespace string) ApplicationStatusMeta {
+	var applicationStatusMeta ApplicationStatusMeta
+	var appConfig corev1alpha2.ApplicationConfiguration
+	if err := c.Get(ctx, client.ObjectKey{Name: applicationName, Namespace: namespace}, &appConfig); err == nil {
+
+		componentName := appConfig.Spec.Components[0].ComponentName
+
+		if component, err := GetComponent(ctx, c, componentName, namespace); err == nil {
+			var workload corev1alpha2.WorkloadDefinition
+			json.Unmarshal(component.Spec.Workload.Raw, &workload)
+
+			var trait corev1alpha2.TraitDefinition
+			traitDefinitionList := ListTraitDefinitionsByApplicationConfiguration(appConfig)
+			if len(traitDefinitionList) != 0 {
+				trait = traitDefinitionList[0]
+			}
```

Comment:
why only check the first one of slice?

Reply:
Fixed, one workload can be attached several traits.